flows/actions/save_contact_field.go

```diff
@@ -55,26 +55,7 @@ func (a *SaveContactField) Execute(run flows.FlowRun, step flows.Step) error {
 		run.AddError(step, err)
 	}
 
-	// if this is either name or language, we save directly to the contact
```

Comment:
This is unrelated - am assuming this stuff isn't meant to be here since update_contact is the action that updates language